### ISSUE TITLE
TFLite support with Google Edge TPU acceleration

### ISSUE THREAD

**Issue by even-wei** [triton-inference-server/server#2276]
**Is your feature request related to a problem? Please describe.**
Triton does not support TFLite and Google Edge TPU yet.

**Describe the solution you'd like**
We need a TFLite backbone and support Google Edge TPU.

**Describe alternatives you've considered**
I had successfully performed inference with Edge TPU using custom python backend.

**Comment by dzier** [comment]
We appreciate your interest. We have marked this as an enhancement request internally.

**Comment by whateverforever** [comment]
I'd add a point about relevance: Supporting tflite is particularly useful, because virtually all optimizations on tensorflow graphs (quantization, pruning) only work when converting the model to tflite. Thanks for marking this issue :+1:

**Comment by wardVD** [comment]
Is there any roadmap for when this feature could be implemented?

**Comment by rgov** [comment]
I looked into implementing it in ONNXRuntime, so Triton would get it "for free." [Here's a thread I wrote over at ORT](https://github.com/microsoft/onnxruntime/discussions/12529) (no replies). And I also asked TFLite a relevant question to implementing this [in this thread] (no replies).

I did an experiment to see [which models could be converted from .onnx format to TF Lite format without issues](https://gist.github.com/rgov/7f1b6ce7fe1b5f746fc31b108fc38741) and most models seemed to work (though I didn't attempt any inference to compare output).
